apps/web/src/design-system/table/Table.tsx

```diff
@@ -89,15 +88,20 @@ export function Table({
   const handlePageChange = (pageNumber) => {
     gotoPage(pageNumber - 1);
   };
+
   const getPageCount = () => {
+    if (total <= pageSize) {
+      return 0;
+    }
+
```

Comment:
A small question just wondering, why is this needed if on line 152 we are checking:
`getPageCount() > 1`

Isnt it will only display if there is more than total pages?

Reply:
Yes. Sorry didn't noticed it, after all they are doing same work.